index.js

```diff
@@ -0,0 +1,87 @@
+    live: '#',
+    source: '#',
+  },
+  project3: {
+    title: 'Project Name goes here',
+    img: './images/card-three.svg',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi',
+    stack: ['HTML/CSS', 'Ruby on rails', 'JavaScript'],
+    live: '#',
+    source: '#',
+  },
+  project4: {
+    title: 'Project Name goes here',
+    img: './images/card-four.svg',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi',
+    stack: ['HTML/CSS', 'Ruby on rails', 'JavaScript'],
+    live: '#',
+    source: '#',
+  },
+  project5: {
+    title: 'Project Name goes here',
+    img: './images/card-five.svg',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi',
+    stack: ['HTML/CSS', 'Ruby on rails', 'JavaScript'],
+    live: '#',
+    source: '#',
+  },
+  project6: {
+    title: 'Project Name goes here',
+    img: './images/card-six.svg',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi',
+    stack: ['HTML/CSS', 'Ruby on rails', 'JavaScript'],
+    live: '#',
+    source: '#',
+  },
+};
+
+const btns = document.querySelectorAll('.btn');
+btns.forEach((b) => {
+  b.addEventListener('click', () => {
+    const parent = b.parentNode;
+    const div = document.createElement('div');
+    div.className = 'popup';
+    div.innerHTML = `<div class="modalbox">
+        <img id="cross" src="./images/ic_cross.svg" alt="ic_cross">
+        <p>${projects[b.id].title}</p>
```

Comment:
- [ ] As the project name is a title, it is recommended to use `<hx></hx>` for titles, not `<p></p>` tag, to increase HTML accessibility.
  - Such as you can consider using `<h2></h2>` or `<h3></h3>` tag.

- [ ] Kindly follow the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9) to style the title.
***Your Design***
![Screenshot from 2023-01-06 05-08-46](https://user-images.githubusercontent.com/60083437/210897400-4fc738b6-98bf-43f0-9b57-01dc38ae8c2a.png)
***Figma Design***
![Screenshot from 2023-01-06 05-07-58](https://user-images.githubusercontent.com/60083437/210897337-0d208ac4-9b10-40e4-9b0d-93930ff5e7da.png)